dev/src/v1/firestore_client_config.json

```diff
@@ -2,42 +2,37 @@
   "interfaces": {
     "google.firestore.v1.Firestore": {
       "retry_codes": {
+        "non_idempotent": [],
         "idempotent": [
           "DEADLINE_EXCEEDED",
-          "INTERNAL",
```

Comment:
We need to fix this for the main client for sure.

Reply:
FWIW, the `googleapis` definition is correct: https://github.com/googleapis/googleapis/blob/master/google/firestore/v1/firestore_grpc_service_config.json#L39

Reply:
These seem to be default configurations that are unused. Resolving.